src/charts/stacked-area.js

```diff
@@ -536,35 +537,57 @@ define(function(require){
          * @private
          */
         function drawStackedAreas() {
+            let series;
+
             // Creating Area function
             area = d3Shape.area()
                 .curve(d3Shape.curveMonotoneX)
                 .x( ({data}) => xScale(data.date) )
                 .y0( (d) => yScale(d[0]) )
                 .y1( (d) => yScale(d[1]) );
 
-            let series = svg.select('.chart-group').selectAll('.layer')
-                .data(layersInitial)
-                .enter()
-              .append('g')
-                .classed('layer-container', true);
-
-            series
-              .append('path')
-                .attr('class', 'layer')
-                .attr('d', area)
-                .style('fill', ({key}) => categoryColorMap[key]);
-
-            // Update
-            svg.select('.chart-group').selectAll('.layer')
-                .data(layers)
-                .transition()
-                .delay( (_, i) => areaAnimationDelays[i])
-                .duration(areaAnimationDuration)
-                .ease(ease)
-                .attr('d', area)
-                .style('opacity', areaOpacity)
-                .style('fill', ({key}) => categoryColorMap[key]);
+            if (isAnimated) {
+                series = svg.select('.chart-group').selectAll('.layer')
```

Comment:
In this case they are slightly different (layersInitial vs layers), so I cannot extract it.